components/create_swagger_config.py

```diff
@@ -102,11 +96,13 @@ def plural_obj_name_to_singular(obj_name, post_fix='', post_fix_used=None):
     acronyms = ['Ads', 'Nis']  # list of acronyms that end in 's'
     # if it's two 'ss' on the end then don't remove the last one
     if (obj_name not in acronyms and obj_name[-1] == 's' and
-            obj_name[-2] != 's'):
+            obj_name[-2] != 's' and not obj_name.endswith('tus')):
         # if container object ends with 's' then trim off the 's'
         # to (hopefully) create the singular version
         if obj_name[-3:] == 'ies':
             one_obj_name = obj_name[:-3].replace('_', '') + 'y'
+        elif (obj_name[-4:] == 'ches' or obj_name[-5:] == 'iases'):
```

Comment:
Will these naming corrections affect the naming of classes? Will this be a breaking change for users of the SDK?

Reply:
@cbrainerd Yes, these naming changes will result in some breaks in the existing SDK API. The class names should not be impacted, but some method names will be impacted. The three names that were corrected were alias, patch, and status, although "status" only appeared as "statu" in the config. Thus, the following methods will change:

- `ProtocolsApi.create_nfs_aliase` -> `ProtocolsApi.create_nfs_alias`
- `ProtocolsApi.delete_nfs_aliase` -> `ProtocolsApi.delete_nfs_alias`
- `ProtocolsApi.get_nfs_aliase` -> `ProtocolsApi.get_nfs_alias`
- `ProtocolsApi.update_nfs_aliase` -> `ProtocolsApi.update_nfs_alias`
- `UpgradeApi.create_cluster_patch_patche` -> `UpgradeApi.create_cluster_patch_patch`
- `UpgradeApi.delete_cluster_patch_patche` -> `UpgradeApi.delete_cluster_patch_patch`
- `UpgradeApi.get_cluster_patch_patche` -> `UpgradeApi.get_cluster_patch_patch`

Reply:
Thanks for the details :+1: